### ISSUE TITLE
BUG: [I'm having trouble installing]

### ISSUE THREAD

**Issue by Cosplayeridol** [wAIfu-DEV/w-AI-fu_v2#26]
**Describe the bug**
I'm having trouble installing
**To Reproduce**

**Expected behavior**
I want to solve installation problems

**Screenshots**
![image](https://github.com/wAIfu-DEV/w-AI-fu_v2/assets/85826393/40883400-29aa-47f6-9e27-6037fae92c7b)

**Additional context**

**Comment by Malshaghab** [comment]
I faced similar issue, and it seems that the ai doesn't fid the node.js even though it is installed.

I solved from here: 
https://stackoverflow.com/questions/27864040/fixing-npm-path-in-windows-8-and-10

If you installed Node.js, 
Steps:
1-go to control panel
2-Type Environment Variables
3- under system, click on Edit the system environment variables (as administrator.  (you will find Edit the system environment variables next to a shield icon)
4-a window opened, now find the box Environment Variables and click it. 
5-a new window opened, in the bottom you will find system variables, search for "Path.." click it then click "Edit..."
6-a new window opened. Click "New" and name it as
"C:\Program Files\nodejs"
or 
";C:\Program Files\nodejs"
I don't know which one of them worked but I added both.

The Error should be gone, but you will need to re-open the Ai

**Comment by Cosplayeridol** [comment]
Is it possible to provide an integration package, which may be easier for novices to use?

**Comment by Cosplayeridol** [comment]
![image](https://github.com/wAIfu-DEV/w-AI-fu_v2/assets/85826393/37e18aba-d903-4a57-9198-81b125a98e5d)
I worked on it for a day, and with the help of Google, my node was running perfectly, including the environment variables that were set. In fact, through the command of the bat script, it always looked for the wrong node path. I tested the node-related cmd myself. The commands are all functioning normally.
In addition, I asked gpt how to modify the scripting language, but it was not easy to use. My requirement is to change, for example, the path to look for nodes in the scripting language, and for the script to use the cnpm install command instead of the npm install command to install (due to proxy restrictions, I use the cnpm command to download very quickly)
I have installed some dependencies through the cmd command box, but there are still some that I don’t know how to install. I can only understand part of the script code.

**Comment by Cosplayeridol** [comment]
![image](https://github.com/wAIfu-DEV/w-AI-fu_v2/assets/85826393/bbb2e82a-f604-4372-b212-5bc5536b5463)
Now the software runs like this

**Comment by wAIfu-DEV** [comment]
For the first image ot looks like the script is run in administrator mode, when you right-click the INSTALL.bat file, do you have multiple options to open it without "Run as administrator" ?

**Comment by Cosplayeridol** [comment]
Of course, I have been running it in administrator mode. As for my suggestions, I hope this can make the project reach more people. For example, the integration package can allow more people to get started quickly without many unknowns during the installation process. mistake. Of course, it would be great if you could guide me on how to modify the script file so that I can successfully install and get started quickly. There are many users of ai vtuber in the Mandarin area, such as me, but they have not yet come into contact with such an excellent project, which is as effective as neuro. Perhaps the lack of effective LLM and TTS in Mandarin is also a big factor, but some of the technologies used in this project may be a breakthrough.

**Comment by stumbleii** [comment]
Bro , have you solved this problem?  I
![Snipaste_2024-08-18_12-11-02](https://github.com/user-attachments/assets/49548df8-7773-440a-bd8a-0562821497f7)
 have same issue,  the interesting thing is we all run this program in  Chinese environment , Maybe that's why the program went wrong ,Could you help me?
